great_expectations/datasource/fluent/pandas_datasource.py

```diff
@@ -48,15 +48,13 @@
 from great_expectations.datasource.fluent.sources import (
     DEFAULT_PANDAS_DATA_ASSET_NAME,
 )
-from great_expectations.util import NotImported
+from great_expectations.optional_imports import sqlalchemy
```

Comment:
We didn't edit this file in mob typing today but it imported `NotImported` from the old location so I updated it to use our new agreed upon pattern.